Misc/NEWS.d/next/Library/2018-05-13-14-19-13.bpo-33461.uteyf-.rst

```diff
@@ -0,0 +1,2 @@
+json.loads *encoding* keyword which was deprecated and ignored since Python
+3.1 has been removed
```

Comment:
Markup string can be used for linking. How about the below:

    Removed the *encoding* keyword from :func:`json.loads`, it was deprecated
    and ignored since Python 3.1. Patch by Matthias Bussonnier.